atlas_densities/densities/fitting.py

```diff
@@ -573,6 +612,23 @@ def _get_group_names(region_map: "RegionMap", group_ids_config: dict) -> dict[st
     }
 
 
+def _get_group_region_names(groups):
+    """
+    Create a set of the union of AIBS region names that belong to the given groups.
+
+    Args:
+        groups: dictionary from group name to list of region name.
+
+    Returns:
+        A list of region names.
+    """
+
+    group_regions_names: Set[str] = set()
+    for v in groups.values():
+        group_regions_names = group_regions_names.union(v)
+    return list(group_regions_names)
```

Comment:
using `import itertools as it`:
`group_regions_names = set(it.chain.from_iterable(groups.values()))`